europe/README.md

```diff
@@ -1 +1,5 @@
 # Europe
+
+## Germany
+
+  * [Würzburg](germany/wuerzburg.md)
```

Comment:
Tim's comment on adding a new city is accidently removed as I pressed 'Resolve'. Sorry.

Reply:
Hah, find it by clicking 'Show Resolved Treads'.